site.yml

```diff
@@ -347,3 +347,13 @@
       tags: ['openstack', 'setup', 'openstack-network']
       when: neutron.enable_external_interface|default('false')|bool
   environment: "{{ env_vars|default({}) }}"
+
+# This is the last role because ceph health checks can take up to 2 hours
+- name: ceph update
+  hosts: ceph_monitors:ceph_osds:controller:compute
```

Comment:
I still don't know why this role has to run on controller or compute. Neither of those nodes get any ceph services, there shouldn't be a need to restart anything there. The ceph config files do exist there for use by other services, but I believe those are all executed and read the config at run time every time.

Please help me understand why we need to run this role on anything other than monitors and osds.